cli/src/commands/logs.ts

```diff
@@ -25,18 +31,19 @@ $ ${Command.cliBinName} logs --since 2m --follow`,
 
   public static flags = {
     ...Command.flags,
-    limit: flags.minMaxInt({
+    limit: flags.integer({
       char: 'l',
       description: 'Limit number of entries shown (cannot exceed 1000).',
       default: 500,
-      min: 1,
-      max: 1000,
+      // min: 1,
```

Comment:
Why?  Didn't we just add documentation with `min`, `max` on https://github.com/binaris/shiftjs/pull/143/files#diff-ccac2b54f1b9872d1d8933d39a4514d1R80?

Reply:
we dropped it until minMaxInt will support defaults properly